### PR TITLE
Updates to MZ crop type and LULC mapping

### PR DESCRIPTION
**Pull request by gabrieltseng**
- the `max_matches` in the dataset config was too low. Our range of dates covered ~7.5 months, but the max matches was 6. I lengthened the time to 8 months in `create_windows_for_lulc.py` and updated the dataset config to reflect this

- slightly increased the proportion of points in the train set vs. the val set. I did this because I observed that even with 0.8 / 0.2 splits, the clustered nature of the crop type labels meant that some crops still had many (up to 50%) of points in the val set (e.g. rice)

- increase the patch size to 4. This massively speeds up inference without penalizing performance

- increase batch size. We originally had it low due to memory issues, but somewhere along the line they got resolved and an 80Gb A100 supports a batch size of 64.

- update the `dataset.json` to not export S1, since we don't use it

### CODE REVIEW

File: olmoearth_projects/projects/mozambique_lulc/update_train_test_split.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    )
+    parser.add_argument("--crop_type", action="store_true", default=False)
+    args = parser.parse_args()
+
+    if args.splitter == "spatial":
+        splitter = SpatialDataSplitter(
+            train_prop=0.9, val_prop=0.1, test_prop=0.0, grid_size=32
+        )
+    elif args.splitter == "random":
+        splitter = RandomDataSplitter(train_prop=0.9, val_prop=0.1, test_prop=0.0)
```
Comment (yawenzzzz) [inline olmoearth_projects/projects/mozambique_lulc/update_train_test_split.py]: Suggestion: add ValueError if splitter is not `spatial` or `random`

**Review by yawenzzzz — APPROVED** [review]
LGTM!
